.github/workflows/ci.yml

```diff
@@ -25,8 +25,13 @@ jobs:
   lingua-franca:
     runs-on: ubuntu-latest
     steps:
+      - name: Check out reactor-ts repository
+        uses: actions/checkout@v3
+        with: 
+          path: "reactor-ts-external"
```

Comment:
```suggestion
          path: "reactor-ts"
```

Reply:
Adding "external" is confusing to me because it's actually using the local version of `reactor-ts`...